src/fontra/views/editor/editor.js

```diff
@@ -458,23 +458,25 @@ export class EditorController {
     const onPointerMove = (event) => {
       if (sidebarResizing) {
         let width;
+        let cssProperty;
         if (growDirection === "left") {
           width = initialWidth + (initialPointerCoordinateX - event.clientX);
-        } else if (growDirection === "right") {
+          cssProperty = "--sidebar-content-width-right";
+        } else {
           width = initialWidth + (event.clientX - initialPointerCoordinateX);
+          cssProperty = "--sidebar-content-width-left";
         }
         width = clamp(width, 200, 500);
-        sidebarResizing.style.width = `${width}px`;
+        document.querySelector(":root").style.setProperty(cssProperty, `${width}px`);
```

Comment:
Can we use `document.documentElement` instead of `document.querySelector(":root")`, or would that be a different element, or a less preferred style?

Reply:
We may move them to `#editor-container`. Is that what you mean?

Reply:
No, I literally meant to ask about `document.querySelector(":root")` vs. `document.documentElement`. Are they the same, and if yes, should we use the former instead?

Reply:
Ok. I think it's ok, we can use documentElement. I will send a patch.